IFC4x3/Properties/c/CorrosionTreatmentBolts_3s$3Dm4hP86vHnuYbK$pjd/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Propose removing this as it can be modelled with IfcMechanicalFastener.BOLT with its own CorrosionTreatment.

Reply:
... or IfcGroundRrinforcementElement/BOLT